recipe/meta.yaml

```diff
@@ -8,25 +8,30 @@ package:
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/magika-{{ version }}.tar.gz
   sha256: e3dd22c73936630b1cd79d0f412d6d9a53dc99ba5e3709b1ac53f56bc998e635
+  skipe: true  #[py<38]
 
 build:
   entry_points:
     - magika-python-client = magika.cli.magika_client:main
     - magika = magika.cli.magika_rust_client_not_found_warning:main
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 0 
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python
     - hatchling
```

Comment:
```suggestion
    - maturin >=1.7,<2.0
```

Reply:
https://github.com/google/magika/blob/ef68f6cba046f2d352863c2952596cd8b2df644e/python/pyproject.toml#L66C21-L66C31

Reply:
It seems switching to `maturin, causes a build fail for all platforms. I wonder if this might be an upstream issue.